modules/dxp/apps/scim/scim-configuration-web/src/main/java/com/liferay/scim/configuration/web/internal/portlet/action/SaveScimConfigurationMVCActionCommand.java

```diff
@@ -170,11 +172,31 @@ else if (Objects.equals(cmd, "reset")) {
 			ActionableDynamicQuery actionableDynamicQuery =
 				_expandoValueLocalService.getActionableDynamicQuery();
 
+			ExpandoColumn userScimClientExpandoColumn =
```

Comment:
Hey @cdbm, move the expandoColumn variables closer to their usage and put the condition related to the `columnId` before the `data` condition just to follow the alphabetical order. At the end, the code should be like this

```
actionableDynamicQuery.setAddCriteriaMethod(
				dynamicQuery -> {
					ExpandoColumn userScimClientExpandoColumn =
						_expandoColumnLocalService.getColumn(
							themeDisplay.getCompanyId(), User.class.getName(),
							"CUSTOM_FIELDS", "scimClientId");

					ExpandoColumn userGroupScimClientExpandoColumn =
						_expandoColumnLocalService.getColumn(
							themeDisplay.getCompanyId(),
							UserGroup.class.getName(), "CUSTOM_FIELDS",
							"scimClientId");

					Property columnProperty = PropertyFactoryUtil.forName(
						"columnId");

					dynamicQuery.add(
						columnProperty.in(
							new long[] {
								userScimClientExpandoColumn.getColumnId(),
								userGroupScimClientExpandoColumn.getColumnId()
							}));

					Property dataProperty = PropertyFactoryUtil.forName("data");

					dynamicQuery.add(dataProperty.eq(scimClientId));
				});

```

After this change, please check the test results and if they are good, forward this to BChan

Thanks